topics/computational-chemistry/tutorials/zauberkugel/tutorial.md

```diff
@@ -0,0 +1,311 @@
+>
+>
+> 1. Create a new file using the Galaxy upload manager, with the following contents. Make sure to select the datatype (with **Type**) as `smi`. This step is essential, as Galaxy does not automatically detect the datatype for SMI files.
+>
+>    ```
+>    C[C@@]12[C@@H]([C@@H](C[C@@H](O1)N3C4=CC=CC=C4C5=C6C(=C7C8=CC=CC=C8N2C7=C53)CNC6=O)NC)OC	staurosporine
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_create_new_file.md format="smi" %}
+>
+> > ### {% icon tip %} Tip: SMILES generation
+> >
+> > A SMILES string can automatically be generated from a ligand name or 2D structure with a desktop molecule editor such [ChemDraw®](https://perkinelmerinformatics.com/products/research/chemdraw/) and [Marvin®](https://chemaxon.com/products/marvin), or with web-based molecule editors such as [PubChem Sketcher](https://pubchem.ncbi.nlm.nih.gov//edit3/index.html) and [ChemDraw® JS](https://chemdrawdirect.perkinelmer.cloud/js/sample/index.html). Moreover, the pre-computed SMILES strings of a large number of bioactive compounds can be retrieved from chemical databases such as [PubChem](https://pubchem.ncbi.nlm.nih.gov/). e.g.
+> >
+> >    ```
+> >    https://pubchem.ncbi.nlm.nih.gov/compound/44259#section=Isomeric-SMILES&fullscreen=true
+> >    ```
+> >
+> {: .tip}
+>
+{: .hands_on}
+
+> ### {% icon question %} Question
+>
+> Why do we specifically use a so-called isomeric SMILES string?
+>
+> > ### {% icon solution %} Solution
+> >
+> > Staurosporine is a chiral molecule possessing four chiral centers. The SMILES notation allows the specification of configuration at tetrahedral centers and double bond geometry, by marking atoms with `@` or `@@`. These are structural features that cannot be specified by connectivity alone, and therefore SMILES which encode this information are termed isomeric SMILES. A notable feature of these rules is that they allow rigorous partial specification of chirality.
+> >
+> {: .solution}
+>
+{: .question}
+
+
+# Pre-processing
+
+Prior to pharmacophore alignment, the predominant ionization state(s) of the query ligand as well as its 3D conformers should be generated. Also, the pharmacophore dataset will be split into a collection of individual pharmacophore files.
+ 
+## Ligand hydration
+
+More often than not, the bioactive form of a compound is its predominant form at physiological pH (7.4). In this step, we predict the most probable ionization state(s) of the query ligand at pH 7.4 with the cheminformatics toolkit OpenBabel ({% cite OBoyle2011 %}).
+
+> ### {% icon hands_on %} Hands-on 4: Add hydrogen atoms
+>
+> 1. {% tool [Add hydrogen atoms](toolshed.g2.bx.psu.edu/repos/bgruening/openbabel_addh/openbabel_addh/3.1.1+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Molecular input file"*: `output` (Input dataset)
+>    - *"Add hydrogens to polar atoms only (i.e. not to carbon atoms)"*: `Yes`
+>
+{: .hands_on}
+
+> ### {% icon question %} Question
```

Comment:
Could you include a figure at some point in the tutorial depicting the structure of the compound? I think it would make these kind of discussions much more straightforward to follow.

Reply:
I updated Figure 2, by adding a 2D depiction of staurosporine next to its 3D depiction.